add info dir section and entry to texdraw and epspdf
---
Index: texlive-base-2011.20120216/texmf/doc/info/texdraw.info
===================================================================
--- texlive-base-2011.20120216.orig/texmf/doc/info/texdraw.info	2006-01-03 09:00:18.000000000 +0900
+++ texlive-base-2011.20120216/texmf/doc/info/texdraw.info	2012-02-17 00:14:24.401360605 +0900
@@ -14,6 +14,11 @@
 entire resulting derived work is distributed under the terms of a
 permission notice identical to this one.
 
+INFO-DIR-SECTION TeX
+START-INFO-DIR-ENTRY
+* TeXdraw: (texdraw).      A system for producing PostScript drawings from TeX.
+END-INFO-DIR-ENTRY
+
 
 File: texdraw,  Node: Top,  Next: Introduction,  Prev: (dir),  Up: (dir)
 
Index: texlive-base-2011.20120216/texmf/doc/info/epspdf.info
===================================================================
Binary files texlive-base-2011.20120216.orig/texmf/doc/info/epspdf.info	2011-02-27 22:21:24.000000000 +0900 and texlive-base-2011.20120216/texmf/doc/info/epspdf.info	2012-02-17 00:14:24.397360572 +0900 differ
